ppx/html.ml

```diff
@@ -15,558 +17,1859 @@ type eventType =
+  ; {tag= "picture"; attributes= domAttributes & attributesHTML}
+  ; {tag= "pre"; attributes= domAttributes & attributesHTML}
+  ; { tag= "progress"
+    ; attributes= domAttributes & attributesHTML & progressHTMLAttributes }
+  ; {tag= "q"; attributes= domAttributes & attributesHTML & quoteHTMLAttributes}
+  ; {tag= "rp"; attributes= domAttributes & attributesHTML}
+  ; {tag= "rt"; attributes= domAttributes & attributesHTML}
+  ; {tag= "ruby"; attributes= domAttributes & attributesHTML}
+  ; {tag= "s"; attributes= domAttributes & attributesHTML}
+  ; {tag= "samp"; attributes= domAttributes & attributesHTML}
+  ; { tag= "script"
+    ; attributes= domAttributes & attributesHTML & scriptHTMLAttributes }
+  ; {tag= "section"; attributes= domAttributes & attributesHTML}
+  ; { tag= "select"
+    ; attributes= domAttributes & attributesHTML & selectHTMLAttributes }
+  ; { tag= "slot"
+    ; attributes= domAttributes & attributesHTML & slotHTMLAttributes }
+  ; {tag= "small"; attributes= domAttributes & attributesHTML}
+  ; { tag= "source"
+    ; attributes= domAttributes & attributesHTML & sourceHTMLAttributes }
+  ; {tag= "span"; attributes= domAttributes & attributesHTML}
+  ; {tag= "strong"; attributes= domAttributes & attributesHTML}
+  ; { tag= "style"
+    ; attributes= domAttributes & attributesHTML & styleHTMLAttributes }
+  ; {tag= "sub"; attributes= domAttributes & attributesHTML}
+  ; {tag= "summary"; attributes= domAttributes & attributesHTML}
+  ; {tag= "sup"; attributes= domAttributes & attributesHTML}
+  ; { tag= "table"
+    ; attributes= domAttributes & attributesHTML & tableHTMLAttributes }
+  ; {tag= "tbody"; attributes= domAttributes & attributesHTML}
+  ; {tag= "td"; attributes= domAttributes & attributesHTML & tdHTMLAttributes}
+  ; {tag= "template"; attributes= domAttributes & attributesHTML}
+  ; { tag= "textarea"
+    ; attributes= domAttributes & attributesHTML & textareaHTMLAttributes }
+  ; {tag= "tfoot"; attributes= domAttributes & attributesHTML}
+  ; {tag= "th"; attributes= domAttributes & attributesHTML & thHTMLAttributes}
+  ; {tag= "thead"; attributes= domAttributes & attributesHTML}
+  ; { tag= "time"
+    ; attributes= domAttributes & attributesHTML & timeHTMLAttributes }
+  ; {tag= "title"; attributes= domAttributes & attributesHTML}
+  ; {tag= "tr"; attributes= domAttributes & attributesHTML}
+  ; { tag= "track"
+    ; attributes= domAttributes & attributesHTML & trackHTMLAttributes }
+  ; {tag= "u"; attributes= domAttributes & attributesHTML}
+  ; {tag= "ul"; attributes= domAttributes & attributesHTML}
+  ; {tag= "var"; attributes= domAttributes & attributesHTML}
+  ; {tag= "video"; attributes= domAttributes & videoHTMLAttributes}
+  ; {tag= "wbr"; attributes= domAttributes & attributesHTML}
+  ; { tag= "webview"
+    ; attributes= domAttributes & attributesHTML & webViewHTMLAttributes } ]
+
+let svgElements =
+  [ {tag= "svg"; attributes= domAttributes & svgAttributes & ariaAttributes}
```

Comment:
As all elements "inherit" from the same groups of attrs, might be worth having just a list of strings with the tags and a map function that builds the full record with `tag` and `attributes` fields.